src/sum_up.py

```diff
@@ -0,0 +1,13 @@
+"""This is the module for summing up numbers within a random string."""
+import re
+
+
+def sum_from_string(string):
+    """The function sums up the numbers in a string."""
+    if any(i.isdigit() for i in string):
```

Comment:
No need to even make this check here, since your regex will just return an empty list if no digits are found.